install/src/main/resources/aspects.bzl

```diff
@@ -494,6 +497,8 @@ def _bsp_target_info_aspect_impl(target, ctx):
         java_target_info = java_target_info,
         java_toolchain_info = java_toolchain_info,
         java_runtime_info = java_runtime_info,
+        env = rule_attrs.env,
```

Comment:
I think it can fail if there is no `env` attribute, please take a look at https://github.com/JetBrains/bazel-bsp/blob/f77d734f631b8d9875c0a5f842504a236785be52/install/src/main/resources/aspects.bzl#L587=

Reply:
```
env = dict()
if hasattr(rule_attrs, "env") {
  env = rule_attrs.env
}
```

Reply:
u can just use `getattr`

Reply:
https://docs.bazel.build/versions/main/skylark/lib/globals.html#getattr